src/libAtomVM/mailbox.h

```diff
@@ -245,10 +250,11 @@ bool mailbox_peek(Context *ctx, term *out);
  * @brief Remove next message from mailbox.
  *
  * @details Discard a term that has been previously queued on a certain process
- * or driver mailbox. To be called from the process only.
+ * or driver mailbox. To be called from the process only. Term messages are actually
+ *  added as fragments to the heap and will be gone at next GC.
  * @param mbx the mailbox to remove next message from.
  */
-void mailbox_remove(Mailbox *mbx);
+void mailbox_remove(Context *ctx);
```

Comment:
These changes are a little strange because they take a context, not a Mailbox, which is kind of against the patterns we have been using.  Perhaps these should be moved into the context module and named accordingly.

Reply:
Eventually, I changed the signature of this function and renamed it accordingly.